core/src/consensus.rs

```diff
@@ -62,16 +62,12 @@ pub const SWITCH_FORK_THRESHOLD: f64 = 0.38;
 
 #[derive(Default, Debug, Clone)]
 pub struct StakeLockout {
-    lockout: u64,
```

Comment:
@carllin Oh, the CI passed. So, I'm sure those removed code is really not needed. (Thanks for this [tip](https://github.com/solana-labs/solana/pull/10719#discussion_r443041006))

So, how should I proceed this tiny cleaning-up PR? I'm still new to these code area. I mean, ...
1. should `StakeLockout` be just reverted to merely `type Stake = u64;` because the `StakeLockout` was a thing of past?
2. should `StakeLockout` be reshaped to something like `struct Stake(u64)` or `struct Stake { stake: u64 }`, anticipating some more fields are added to it?

Reply:
@ryoqun, I think it makes sense to just make Stake = u64, since there are no other fields. I don't anticipate any new fields as of now.